nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -55,14 +55,6 @@ in {
         description = "Enable Gnome 3 desktop manager.";
       };
 
-      sessionPath = mkOption {
```

Comment:
Is there a way to redirect users to the new option (extraSessionFilePackages?) instead?

Reply:
It serves a completely different purpose. The replacement to `sessionPath` is fixing packages.